lib/autoproj/ops/configuration.rb

```diff
@@ -260,19 +290,23 @@ def load_and_update_package_sets(root_pkg_set, only_local = false)
                 queue_auto_imports_if_needed(queue, pkg_set, root_pkg_set)
             end
 
-            cleanup_remotes_dir(package_sets)
+            cleanup_remotes_dir(package_sets, required_remotes_dirs)
             cleanup_remotes_user_dir(package_sets)
```

Comment:
My problem with it is that it goes the long route (implicitely through the package sets) to determine what are irrelevant symlinks when there is an easy way to go the short route (add it to a set whenever call the symlink update method)